### PR TITLE
added mailroom.py --part 1

### PR DESCRIPTION
**Pull request by AlyssaHong**
added mailroom.py --part 1

### CODE REVIEW

File: students/AlyssaHong/lesson03/mailroom.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        print('{:<20}   ${:>10,.2f}   {:>10}   ${:>10,.2f}'.format\
+             (list_donor_name[i],list_total_given[i],list_num_gifts[i],list_avg_gift[i]))
+    print('{}'.format("-"*63))
+
+def quit():
+    print("Quit current task!")
+
+
+def main():
+    print("Choose the number(1,2,3) from a menu:(1)Send a Thank you (2)Creat a Report (3)quit")
```
Comment (charlesrobison) [inline students/AlyssaHong/lesson03/mailroom.py]: Typo - (2)Create a Report

File: students/AlyssaHong/lesson03/mailroom.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    if choice_action == 1:
+        send_thanks()
+    elif choice_action == 2:
+        creat_report()
+    elif choice_action == 3:
+        quit()
+
+
+if __name__ == '__main__':
+    main()
```
Comment (charlesrobison) [inline students/AlyssaHong/lesson03/mailroom.py]: As written, the program closes after each action.  You'll want it to continue until the user selects quit.

File: students/AlyssaHong/lesson03/mailroom.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            (list_donor_name[i],list_total_given[i],list_num_gifts[i],list_avg_gift[i]))
+
+
+def create_report():
+    print('{:<20} | {:^10} | {:^10} | {:^10}'.format(*list_col))
+    print('{}'.format("-"*63))
+    for i in range(len(list_donor_name)):
+        print('{:<20}   ${:>10,.2f}   {:>10}   ${:>10,.2f}'.format\
+             (list_donor_name[i],list_total_given[i],list_num_gifts[i],list_avg_gift[i]))
+    print('{}'.format("-"*63))
```
Comment (natasha-aleksandrova) [inline students/AlyssaHong/lesson03/mailroom.py]: You are missing a sorting functionality for the report; please review the instructions and add that in.

File: students/AlyssaHong/lesson03/mailroom.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        if donor_name in list_donor_name:
+            i = list_donor_name.index(donor_name)
+            print(donor_name, list_donor_name[i])
+            if list_donor_name[i] in list_donor_name:
+                list_total_given[i] = list_total_given[i] + new_gift
+                print(list_total_given[i])
+                list_num_gifts[i]  = list_num_gifts[i] + 1
+                print(list_num_gifts[i])
+                list_avg_gift[i] = list_total_given[i]/list_num_gifts[i]
+        else:
```
Comment (natasha-aleksandrova) [inline students/AlyssaHong/lesson03/mailroom.py]: line 25 and line 16 should be de-dented; user should be presented to either choose 'list' or add/update donor. Right now you are always forced to type `list`

File: students/AlyssaHong/lesson03/mailroom.py (hunk 2 above)
Comment (natasha-aleksandrova) [reply students/AlyssaHong/lesson03/mailroom.py]: Odd - main loop works fine for me.

**Comment by natasha-aleksandrova** [comment]
@AlyssaHong please add missing sort feature and fix send_thank_you flow per my comment.

Merging.

**Pull request merged**